packages/DateRangePicker/test/DateRangePicker.cypress.js

```diff
@@ -0,0 +1,54 @@
+import moment from "moment";
+
+describe("<DateRangePicker />", () => {
+  const targetStartDate = moment();
+  const targetEndDate = moment(targetStartDate)
+    .add(1, "month")
+    .date(1);
+
+  beforeEach(() => {
+    cy.visitStorybook("forms-daterangepicker-backyard-tests--cypress");
```

Comment:
I added a `getStoryUrlPrefix()` helper to derive this URL from the component name via `storyTree`:
https://github.com/acl-services/paprika/blob/master/packages/DatePicker/test/DatePicker.cypress.js#L11

Alternatively, if you write the story in the new CSF style, you can add in `id` and specify a permalink (in case the component moves to another category).   

https://github.com/acl-services/paprika/blob/master/packages/SidePanel/stories/SidePanel.tests.stories.js#L15